Makefile

```diff
@@ -205,26 +154,20 @@ else
 		--non-interactive \
 		--token ${FIREBASE_TOKEN} \
 		--project ${FIREBASE_ALIAS} \
-		--debug \
-		--json
+		--debug
 endif
 
-
 # All in one command to stage your build to a Firebase 
 # channel on your currently selected project
 # Usage: `make stage STAGE_NAME=foo`
-STAGE_NAME ?= docs
 stage:
```

Comment:
this doesn't need to be addressed now and can be a todo for later on, but it'd be nice to integrate automatic staging on PRs within `build.yml`!

Reply:
Yes, mirror the Dart site? Great idea.

Reply:
Yes!